doi_core.py

```diff
@@ -11,13 +11,10 @@
 # Import the Python libraries                                                                                                   
```

Comment:
Why this file is used for ? Can we remove it ?

Reply:
The file has been removed.  Will show up next time it gets pushed.

{pds-dev-el7.jpl.nasa.gov}/home/qchau/sandbox/pds-doi-service 123 % git rm doi_core.py
rm 'doi_core.py'
{pds-dev-el7.jpl.nasa.gov}/home/qchau/sandbox/pds-doi-service 124 % git commit -m "Remove unused file doi_core.py"
[#28 bd5d48b] Remove unused file doi_core.py